lib/cc/engine/markdownlint.rb

```diff
@@ -33,7 +35,7 @@ def include_paths
         return root unless engine_config.has_key?("include_paths")
 
         markdown_files = engine_config["include_paths"].select do |path|
-          path.end_with?(".md")
+          EXTENSIONS.include?(File.extname("path")) || path.end_with?("/")
```

Comment:
Sigh, yes. I knew I should have written more tests, but writing new fixtures seemed like more trouble than it was worth at the time.